### PR TITLE
Fix #220, normalize task names/descriptions to have both the long and short name in all cases

### LINKED ISSUE #220: Make task names consistent
- Make sure task names are consistent
- Make sure the task name or description contains both the formal and shorted name + AWS and Amazon

### PR DESCRIPTION
**Pull request by hunterwerlla**
## Types of changes

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Added tests
-   [ ] Code cleanup

## Related Issue(s), If Filed

fix #220
## Checklist

-   [ ] I have read the **README** document
-   [ ] I have read the **CONTRIBUTING** document
-   [ ] Local run of `npm run fullBuild` succeeds
-   [ ] I have run tslint against my changes using the root tslint.yaml
-   [ ] My code follows the code style of this project
-   [ ] I have added tests to cover my changes
-   [ ] All new and existing tests passed
-   [ ] A short description of the change has been added to the **CHANGELOG** (CHANGELOG.md) if the change is feature related

## License

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: Tasks/CloudFormationDeleteStack/task.json [hunk 1]
```diff
@@ -2,18 +2,13 @@
     "id": "089bda61-367d-4d7c-a06b-88f7e2f20f22",
     "name": "CloudFormationDeleteStack",
     "friendlyName": "AWS CloudFormation Delete Stack",
-    "description": "Deletes an AWS CloudFormation stack.",
+    "description": "Deletes a AWS CloudFormation stack.",
```
Comment (awschristou) [inline Tasks/CloudFormationDeleteStack/task.json]: The changes to "a AWS" in this PR seem odd to me.

Comment (hunterwerlla) [reply Tasks/CloudFormationDeleteStack/task.json]: I got a little to excited about a vs an, reverted this change

File: Tasks/LambdaInvokeFunction/task.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
     "id": "c9e49f96-6c7d-4420-97f1-5e0dfd816354",
     "name": "LambdaInvokeFunction",
     "friendlyName": "AWS Lambda Invoke Function",
-    "description": "Invokes a AWS Lambda function",
+    "description": "Invokes a Lambda function",
```
Comment (awschristou) [inline Tasks/LambdaInvokeFunction/task.json]: Please restore the AWS prefix

Comment (hunterwerlla) [reply Tasks/LambdaInvokeFunction/task.json]: Done

**Pull request merged**